src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,55 +1,98 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
-
+import { Film } from '../../types/Film';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+
+type Props = {
+  addFilm: (movie: Film) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addFilm }) => {
+  const [inputValue, setValue] = useState('');
+  const [movie, setMovie] = useState<Film>({ Response: 'True' });
+
+  const getDate = () => {
```

Comment:
```suggestion
  const getData = () => {
```
date - дата, data - дані